pkgs/applications/misc/parsec/bin.nix

```diff
@@ -74,6 +78,16 @@ stdenvNoCC.mkDerivation {
     runHook postInstall
   '';
 
+  fixupPhase = ''
+    runHook preFixup
+
+    autoPatchelf $out/bin
+
+    runHook postFixup
+  '';
+
+  dontAutoPatchelf = true;
+
```

Comment:
Please add a comment why this is necessary. Commit messages are nice but it's better to have this sort of stuff in-code.